### DISCUSSION TITLE
Request: A way to distinguish videos & articles

### DISCUSSION THREAD

**Discussion by wfahnestock** [dailydotdev/daily#974] [Feature Requests]
Greetings,

I think it would be a great addition if there was an icon or something when the cards on the feed link to a video/any medium that is not primarily a text article. I looked into doing a PR for this a while back and unfortunately got busy with other things before I could complete it. If my schedule frees up sometime I might take another look into it.

**Comment by idoshamun** [comment]
Hi @wfahnestock,

That's a dup of https://github.com/dailydotdev/daily/discussions/698. Let's keep the discussion there
